gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -113,7 +146,7 @@ end
 -- <a href="https://en.wikipedia.org/wiki/Potentially_visible_set">PVS</a>,
 -- the client will receive it as NULL entity.
 -- @hook
--- @realm server
+-- @realm shared
```

Comment:
shared? Isn't the file executed client only?

Reply:
Good question. This explicit function is executed in the client. However the hook in general is shared. I did not know how to describe this in the right way